From 0838a124c1a616be542a1d7c401298ab91733704 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <pantelis.antoniou@konsulko.com>
Date: Thu, 30 Jul 2020 15:47:51 +0300
Subject: [PATCH 1/2] Makefile: disable gcc9's address-of-packed-member error

This is also picked up in mainline u-boot but in a non-easily
cherry-picked form.

Signed-off-by: Pantelis Antoniou <pantelis.antoniou@konsulko.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 7060da1c5f..94ace1a7a2 100644
--- a/Makefile
+++ b/Makefile
@@ -605,6 +605,7 @@ ifeq ($(shell $(CONFIG_SHELL) $(srctree)/scripts/gcc-stack-usage.sh $(CC)),y)
 endif
 
 KBUILD_CFLAGS += $(call cc-option,-Wno-format-nonliteral)
+KBUILD_CFLAGS += $(call cc-disable-warning, address-of-packed-member)
 
 # turn jbsr into jsr for m68k
 ifeq ($(ARCH),m68k)
-- 
2.17.1

